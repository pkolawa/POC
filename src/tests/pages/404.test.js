import React from "react";
import { shallow } from "enzyme";
import NotFound from "../../pages/404";

test("Should Render Configuration Page", () => {
  const wrapper = shallow(<NotFound/>);
  expect(wrapper).toMatchSnapshot();
});