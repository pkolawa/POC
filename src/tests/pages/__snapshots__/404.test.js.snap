// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Configuration Page 1`] = `
<div>
  <WithStyles(ForwardRef(Typography))
    align="center"
    color="danger"
    variant="h2"
  >
    404 Page Not Found
  </WithStyles(ForwardRef(Typography))>
</div>
`;
